src/Lens/Micro/Aeson/Internal.hs

```diff
@@ -36,34 +30,12 @@ type instance Index Value = Text
 type instance IxValue Value = Value
 
 -- | Can only index into the contents of an `Object`,
--- which is a `HashMap`.
+-- which is a `KM.KeyMap`.
 instance Ixed Value where
   ix i f (Object o) = Object <$> ix (Key.fromText i) f o
   ix _ _ v          = pure v
   {-# INLINE ix #-}
```

Comment:
It this instance still necessary?

Reply:
Yes I suppose the remaining instances are necessary, since bridging the gap between `aeson` and `microlens` is precisely the purpose of this library.